src/bundle/Resources/public/js/scripts/core/custom.dropdown.js

```diff
@@ -131,16 +131,21 @@
             this.sourceInput.dispatchEvent(new CustomEvent(EVENT_VALUE_CHANGED));
         }
 
-        getItemsContainerHeight(isItemsContainerTop) {
+        getItemsContainerHeight(isItemsContainerAbove) {
             const DROPDOWN_MARGIN = 16;
+            const SELECTOR_MODAL = '.modal[aria-modal=true]';
+            const documentElementHeight = document.documentElement.getBoundingClientRect().height;
```

Comment:
You could use `doc` insted `document`

Reply:
It wasn't there in the first place, but why not. Changed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1709/commits/afdbc260465a858e2604d4245bde0cb4c8b6eead.